book/src/building-from-source.md

```diff
@@ -145,8 +145,8 @@ you can configure Helix to show up in the application menu by copying the
 provided `.desktop` and icon files to their correct folders:
 
 ```sh
-cp contrib/Helix.desktop ~/.local/share/applications
-cp contrib/helix.png ~/.icons # or ~/.local/share/icons
+cp contrib/Helix.desktop -- "$HOME/.local/share/applications"
+xdg-icon-resource install --novendor --size 256 contrib/helix.png
```

Comment:
Are there distributions that the `cp` command won't work for? I'd like to keep the `cp` version (at least commented out) if it isn't misleading. Needing to install another package just to install the icon seems pretty heavy to me.

Maybe we can link out to relevant docs (like https://wiki.archlinux.org/title/Icons ?) instead?

Reply:
I agree. For consistency, both cmds could use `cp` and the `xdg-*` cmd would be commented, just in case

Reply:
BTW, when `xdg-icon-resource` runs as `root`, it can `install` system-wide (all users). This is useful for users such as me, that install `hx` on `/usr/local/bin/` (`*.desktop` at `/usr/share/applications/`)